readme.md

```diff
@@ -25,8 +25,8 @@ ___
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_search-bar-airbnb/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_search-bar-airbnb/report/html_report/)
+- [DEMO LINK](https://DariaFesiun.github.io/layout_search-bar-airbnb/)
+- [TEST REPORT LINK](https://DariaFesiun.github.io/layout_search-bar-airbnb/report/html_report/)
```

Comment:
Add these links to the PR description
<img width="968" alt="image" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/64750958/5edc946d-d435-4145-82fb-8ea5de3b6d64">